lib/assertions.js

```diff
@@ -328,6 +328,43 @@ export default class DOMAssertions {
   }
 
   /**
+   * Assert that the text of the [HTMLElement][] or an [HTMLElement][]
+   * matching the `selector` does not contain the given `text`, using the
+   * [`textContent`](https://developer.mozilla.org/en-US/docs/Web/API/Node/textContent)
+   * attribute.
+   *
+   * **Aliases:** `doesNotContainText`, `doesNotHaveTextContaining`
+   *
+   * @param {string} text
+   * @param {string?} message
+   *
+   * @example
+   * assert.dom('#title').doesNotIncludeText('Welcome');
+   */
+  doesNotIncludeText(text, message) {
+    let element = this.findTargetElement();
+    if (!element) return;
+
+    let result = element.textContent.indexOf(text) === -1;
+    let actual = element.textContent;
+    let expected = text;
```

Comment:
`expected = text` does not make sense in this case, right?

Reply:
Good catch! Would you prefer the name `unexpected` or something else?

Reply:
maybe something like `` `does not include "${text}"` ``?

Reply:
I can certainly update the phrasing of the message to `does not include "${text}"`, my question is actually regarding what is passed into `this.pushResult` at the end of this function. Would it be fine to say `this.pushResult({ results, actual, unexpected, message });`?

Reply:
no, because the `pushResult` method explicitly requires an `expected` attribute (see api.qunitjs.com/assert/pushResult)

Reply:
That's what I figured. `this.pushResult({ result, actual, expected: unexpected, message});` would work but seems a little ironic to "expect the unexpected".